src/UploadedFile.php

```diff
@@ -113,6 +113,9 @@ private function setError(int $error): void
         $this->error = $error;
     }
 
+    /**
+     * @param string|mixed $param
```

Comment:
There are a few places where you made this mistake. Never include mixed in a union type. It already includes every other type (apart from void and never). `string|mixed` is just `mixed`.